### ISSUE TITLE
Demon's Souls issues since #8664

### ISSUE THREAD

**Issue by RainbowCookie32** [RPCS3/rpcs3#8689]
## Quick summary
DeS crashes on the tutorial and has vertex explosions after #8664.

## Details
My testing was done only on the Tutorial area, but I've seen other users on Discord reporting the vertex explosions too.
The game consistently crashes on this area when entering the boss' area with `VM: Access violation writing location 0x0 (unmapped memory)`. While checking builds to find what caused the regression, I also found that the same PR introduced vertex explosions on the map. Again, easy to reproduce on the tutorial area.

Taking an RSX Capture of the vertex explosion caused [this](https://cdn.discordapp.com/attachments/272875751773306881/740288803331113071/2020-08-04_16-21-05.mp4) to happen, which is way worse than anything I've seen in normal gameplay.

[Log from the crash](https://github.com/RPCS3/rpcs3/files/5024122/RPCS3.log.gz)
[RSX Capture for the vertex explosion](https://drive.google.com/file/d/17yYKDDlrKoRp9JCfNqxnM_vEDaNtbNM6/view?usp=sharing)

Tested on Manjaro Linux, Ryzen 7 1700, and RX 570 (RADV LLVM 10.0)

**Comment by DemonAk** [comment]
same thing, every time crashes 

F {PPU[0x1000005] Thread (HighGraphics) [0x000353f0]} VM: Access violation writing location 0xff000004 (unmapped memory)

tested with vulkan and opengl
ryzen 3950x, 1080ti. Played on past week and all worked fine but after latest update crashes

**Comment by Nytra** [comment]
Using the "Disable dynamic exposure" patch will fix this crash for now. I also disabled "write color buffers". Although this should only be a temporary solution.

**Comment by dio-gh** [comment]
The crash seems to have been actually caused by #8652, are you sure you bisected properly? The vertex explosions might still be because of #8664 though, needs double checking.

**Comment by kd-11** [comment]
@Nytra So then, does https://github.com/RPCS3/rpcs3/pull/8691 fix it?

**Comment by Nytra** [comment]
> 
> 
> @Nytra So then, does #8691 fix it?

Sorry, I am unsure. All I know is that my game was crashing, I then applied that patch and it stopped crashing.

**Comment by RainbowCookie32** [comment]
Closing in favor of #8692, I goofed up this one D:
